src/util/dump.ts

```diff
@@ -142,3 +142,17 @@ export async function downloadAndSaveModule(zimCreator: ZimCreator, mw: MediaWik
         throw e;
     }
 }
+
+export async function downloadPolyfillModules(zimCreator: ZimCreator, downloader: Downloader) {
+    const { content } = await downloader.downloadContent(encodeURI('https://unpkg.com/webp-hero@0.0.0-dev.26/dist-cjs/polyfills.js'));
```

Comment:
In place of downloading we should have this as an `npm` dependence and read it. Will be more robust.